torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -32,6 +38,18 @@ def __init__(
         self.attribution_method = attribution_method
         self.kwargs = kwargs
 
+    def _get_mask_type(self):
+        "Based on the explainer config, return the mask type."
+        node_mask_type = self.explainer_config.node_mask_type
+        edge_mask_type = self.explainer_config.edge_mask_type
+        if node_mask_type is not None and edge_mask_type is not None:
+            mask_type = 'node_and_edge'
```

Comment:
Could we introduce these three mask types as an Enum? Also in `captum.py`, possibly as a follow-up PR.